app/jobs/group_export_job.rb

```diff
@@ -0,0 +1,11 @@
+class GroupExportJob < ActiveJob::Base
+  queue_as :low_priority
+
+  def perform(group, actor)
+    groups = actor.groups.where(id: group.all_groups)
+    filename = GroupExportService.export_filename_for(group)
+    GroupExportService.export(groups, filename)
```

Comment:
I would have expected
```
filename = GroupExportService.export(group)
```
and then call group.all_groups from within the service

Reply:
The plan was that anyone in a group can export the data. Meaning they should only be able to export data for groups they belong to. So I want to pass in only groups the user belongs to.